_posts/2017-11-20-Running-CAW-with-Singularity.md

```diff
@@ -15,42 +15,40 @@ image:
 
 # Running CAW with Singularity
 
-To begin, I'll explain what our workflow is about, how the cluster we're using is set up, why we wanted to switch to Singularity and finally how we managed it.
-
-## Our workflow
+## The CAW Pipeline
 
 ![alt text](https://raw.githubusercontent.com/SciLifeLab/CAW/master/doc/images/CAW_logo.png "Cancer Analysis Workflow logo")
 
-First of all, a few word about the [Cancer Analysis Workflow](http://opensource.scilifelab.se/projects/caw/) (CAW for short).
+[Cancer Analysis Workflow](http://opensource.scilifelab.se/projects/caw/) (CAW for short) is an analysis pipeline developed for the analysis of tumour : normal pairs.
 It is developed in collaboration with two infrastructures within [Science for Life Laboratory](https://www.scilifelab.se/): [National Genomics Infrastructure](https://ngisweden.scilifelab.se/) (NGI), in The Stockholm [Genomics Applications Development Facility](https://www.scilifelab.se/facilities/ngi-stockholm/) to be precise and [National Bioinformatics Infrastructure Sweden](https://www.nbis.se/) (NBIS).
 
-CAW is a Nextflow workflow designed to analyze tumor/normal pairs.
-Adapted from [GATK Best Practices](https://software.broadinstitute.org/gatk/best-practices/) for the preprocessing of fastq files, then various variant calling tools are used to look for somatic SNVs and small indels ([MuTect1](https://github.com/broadinstitute/mutect/), [MuTect2](https://github.com/broadgsa/gatk-protected/), [Strelka](https://github.com/Illumina/strelka/), [Freebayes](https://github.com/ekg/freebayes/)), ([GATK HaplotyeCaller](https://github.com/broadgsa/gatk-protected/)), for structural variants([Manta](https://github.com/Illumina/manta/)) and for CNVs ([ASCAT](https://github.com/Crick-CancerGenomics/ascat/)).
+CAW uses [GATK Best Practices](https://software.broadinstitute.org/gatk/best-practices/) for the preprocessing of FastQ files, then uses various variant calling tools to look for somatic SNVs and small indels ([MuTect1](https://github.com/broadinstitute/mutect/), [MuTect2](https://github.com/broadgsa/gatk-protected/), [Strelka](https://github.com/Illumina/strelka/), [Freebayes](https://github.com/ekg/freebayes/)), ([GATK HaplotyeCaller](https://github.com/broadgsa/gatk-protected/)), for structural variants([Manta](https://github.com/Illumina/manta/)) and for CNVs ([ASCAT](https://github.com/Crick-CancerGenomics/ascat/)).
```

Comment:
Do we use the GATK BP as intended? I think we might have slightly customized some processes, that's why I was using adapted, but based on might be better.
What do you think?

> However, any workflow that has been significantly adapted or customized, whether for performance reasons or to fit a use case that we do not explicitly cover, should not be called "GATK Best Practices". The correct way to refer to such workflows is "based on" or "adapted from" GATK Best Practices

Reply:
Yeah ok, "CAW is based on GATK Best Practices" then.. Good spot 👍

Reply:
I think we might need to update the README on that point too.